pkg/apis/certmanager/validation/certificate_for_issuer.go

```diff
@@ -95,3 +97,9 @@ func ValidateCertificateForSelfSignedIssuer(crt *v1alpha1.CertificateSpec, issue
 
 	return el
 }
+
+func ValidateCertificateForVenafiIssuer(crt *v1alpha1.CertificateSpec, issuer *v1alpha1.IssuerSpec, specPath *field.Path) field.ErrorList {
```

Comment:
TODO?

Reply:
There is no special validation needed for Venafi certificates, as there are no restrictions on what types of certificates a Venafi instance will issue (unlike ACME) 😄